src/sonic-host-services/scripts/hostcfgd

```diff
@@ -399,6 +399,12 @@ class FeatureHandler(object):
     def set_feature_state(self, feature, state):
         self._feature_state_table.set(feature.name, [('state', state)])
 
+        # Update the feature state in STATE_DB in namespaces in multi-asic platform
+        namespaces = device_info.get_namespaces()
+        for namespace in namespaces:
+            db_conn = DBConnector(STATE_DB, 0, False, namespace);
+            feature_state_tbl = Table(db_conn, 'FEATURE')
+            feature_state_tbl.set(feature.name, [('state', state)])
 
```

Comment:
Check if the hostcfgd starts before orchagent, to make sure there is no timeing issue.

Reply:
wonder if we really want feature table to be in each namespace.

Reply:
I have updated this.